synapse/handlers/auth.py

```diff
@@ -569,6 +574,12 @@ def check_user_exists(self, user_id):
             defer.Deferred: (unicode) canonical_user_id, or None if zero or
             multiple matches
         """
+        self._account_ratelimiter.ratelimit(
+            user_id, time_now_s=self._clock.time(),
+            rate_hz=self.hs.config.rc_login.account.per_second,
```

Comment:
can you pull out these config settings in the constructor please?

Reply:
Nope, because it makes the tests fail, since the test define special config parameters after initiating the handlers and servlets.